pkg/features/httproute.go

```diff
@@ -100,6 +100,9 @@ const (
 
 	// This option indicates support for the name field in the HTTPRouteRule (extended conformance)
 	SupportHTTPRouteNamedRouteRule FeatureName = "HTTPRouteNamedRouteRule"
+
```

Comment:
the features are now defined here: https://github.com/kubernetes-sigs/gateway-api/pull/3990/files#diff-26be395078f25af5157d4f2c9fe1f924022c4332f54ca452b09c2138f523d0d5

we can skip this  (probably this change will be dropped during rebase)